python/lsst/ts/hexrotcomm/base_csc.py

```diff
@@ -155,9 +155,9 @@ class BaseCsc(salobj.ConfigurableCsc):
     -----
     **Error Codes**
 
-    * `lsst.ts.idl.enums.MTHexapod.ErrorCode.CONTROLLER_FAULT`:
+    * `lsst.ts.xml.enums.MTHexapod.ErrorCode.CONTROLLER_FAULT`:
       The low-level controller went to fault state.
-    * `lsst.ts.idl.enums.MTHexapod.ErrorCode.CONNECTION_LOST`:
+    * `lsst.ts.xml.enums.MTHexapod.ErrorCode.CONNECTION_LOST`:
```

Comment:
Same as above, this can be `ErrorCode.CONNECTION_LOST` only.

Reply:
Fixed in the commit: [aca6aed](https://github.com/lsst-ts/ts_hexrotcomm/pull/68/commits/aca6aed921cb2791cad6c4aa62574686dcb127c8). Thanks!